playground/router.ts

```diff
@@ -32,6 +28,6 @@ export function createRouter({ client_id, domain, audience }: any) {
         beforeEnter: authGuard
       }
     ],
-    history: createWebHashHistory()
+    history: createWebHistory()
```

Comment:
Moved the playground to use history mode as I believe that should be the default.